src/Engine/include/OpenLoco/Engine/World.hpp

```diff
@@ -69,11 +69,21 @@ namespace OpenLoco::World
         return (coord >= (World::kTileSize - 1)) && (coord < (World::kMapWidth - World::kTileSize));
     }
 
+    constexpr bool drawableTileCoord(const tile_coord_t coord)
+    {
+        return (coord >= 1) && (coord < (kMapWidth - 1));
```

Comment:
could you fix drawableCoords it should have a min of `(coord >= World::kTileSize)` i checked all existing users